README.md

```diff
@@ -62,29 +62,186 @@ latexmk
 - 在vscode上编译 (LaTex Workshop)：对于使用vscode写tex的用户，添加如下配置，可在vscode上使用XeLaTeX：
 
 ```
-"latex-workshop.latex.tools": [
-    {
-        "name": "xelatex",
-        "command": "xelatex",
-        "args": [
-            "-synctex=1",
-            "-interaction=nonstopmode",
-            "-file-line-error",
-            "%DOC%"
-        ]
-    }
-],
-"latex-workshop.latex.recipes": [
-    {
-        "name": "xelatex -> bibtex -> xelatex*2",
-        "tools": [
-            "xelatex",
-            "bibtex",
-            "xelatex",
-            "xelatex"
-        ]
-    }
-]
+// ======================== LaTeX 设置 BEGIN  ========================
+	// bibtex 格式
+	"latex-workshop.bibtex-format.tab": "tab",
+
+	// 自动编译，全部关闭，当且仅当你认为有需要的时候才会去做编译
+	"latex-workshop.latex.autoBuild.run": "never",
+	"latex-workshop.latex.autoBuild.cleanAndRetry.enabled": false,
+
+	// 设置 latex-workshop 的 PDF　预览程序，external　指的是外部程序
+	//"latex-workshop.view.pdf.viewer": "external",
+	//"latex-workshop.view.pdf.ref.viewer": "external",
+	//"latex-workshop.view.pdf.external.viewer.command": "D:/Users/newton/Documents/Softwares/software_windows/design/SumatraPDF-3.2-64.exe",
+	//"latex-workshop.view.pdf.external.viewer.args": [
+	//	"%PDF%"
+	//],
+
+	// 配置正向、反向搜索：.tex -> .pdf
+	//"latex-workshop.view.pdf.external.synctex.command": "D:/Users/newton/Documents/Softwares/software_windows/design/SumatraPDF-3.2-64.exe",
+	//"latex-workshop.view.pdf.external.synctex.args": [
+		// 正向搜索
+	//	"-forward-search",
+	//	"%TEX%",
+	//	"%LINE%",
+	//	"-reuse-instance",
+		// 反向搜索
+	//	"-inverse-search",
+	//	"\"C:/Program Files/Microsoft VS Code/Code.exe\" \"C:/Program Files/Microsoft VS Code/resources/app/out/cli.js\" -gr %f:%l",
+	//	"%PDF%"
+	//],
+
+	// 这是一些独立的编译选项，可以作为工具被编译方案调用
+	"latex-workshop.latex.tools": [
+		{
+			// Windows 原生安装 TeX Live 2021 的编译选项
+			"name": "Windows XeLaTeX",
+			"command": "xelatex",
+			"args": [
+				"-synctex=1",
+				"-interaction=nonstopmode",
+				"-file-line-error",
+				"-pdf",
+				"%DOCFILE%"
+			]
+		},
+		{
+			// Windows Biber 编译
+			"name": "Windows Bibtex",
+			"command": "bibtex",
+			"args": [
+				"%DOCFILE%"
+			]
+		},
+		{
+			// WSL XeLaTeX 编译一般的含有中文字符的文档
+			"name": "WSL XeLaTeX",
+			"command": "wsl",
+			"args": [
+				"/usr/local/texlive/2021/bin/x86_64-linux/xelatex",
+				"-synctex=1",
+				"-interaction=nonstopmode",
+				"-file-line-error",
+				"-pdf",
+				//"-output-directory=%OUTDIR%",
```

Comment:
Any reason to comment these (L127-128) out? I would either uncomment them or simply remove them.